api/services/Frontend.js

```diff
@@ -6,7 +6,7 @@ throughout the code.
 
 */
 
-const prefix = `${process.env.PROTOCOL}://${process.env.DOMAIN}`
+const prefix = `${process.env.PROTOCOL}://${process.env.FRONTEND_DOMAIN}`
```

Comment:
why did you find this to be necessary? DOMAIN is supposed to be the frontend domain already